src/Symfony/Component/Notifier/Bridge/Mercure/MercureTransport.php

```diff
@@ -91,8 +93,10 @@ protected function doSend(MessageInterface $message): SentMessage
             $sentMessage->setMessageId($messageId);
 
             return $sentMessage;
-        } catch (ExceptionInterface $e) {
+        } catch (HttpExceptionInterface $e) {
             throw new TransportException(sprintf('Unable to post the Mercure message: "%s".', $e->getResponse()->getContent(false)), $e->getResponse(), $e->getCode(), $e);
+        } catch (ExceptionInterface $e) {
+            throw new RuntimeException(sprintf('Unable to post the Mercure message: "%s".', $e->getMessage()), $e->getCode(), $e);
         } catch (\InvalidArgumentException $e) {
             throw new InvalidArgumentException(sprintf('Unable to post the Mercure message: "%s".', $e->getMessage()), $e->getCode(), $e);
```

Comment:
we should remove the double quotes around the message
there is no need for sprintf() here, this will make fabbot happy